flytestdlib/cli/pflags/api/generator.go

```diff
@@ -235,6 +238,108 @@ func discoverFieldsRecursive(ctx context.Context, workingDirPkg string, typ *typ
 			}
 
 			addField(typ, f)
+		case *types.Alias:
```

Comment:
This is mostly a copy/pasta of the named case but making it work with `*types.Alias`.